src/main/java/th/ac/kmitl/soa/group9/taxInvoice/controllers/TaxInvoiceController.java

```diff
@@ -1,14 +1,24 @@
 package th.ac.kmitl.soa.group9.taxInvoice.controllers;
 
 import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestMapping;
+import th.ac.kmitl.soa.group9.taxInvoice.models.ExchangedDocument;
 
 @Controller
 public class TaxInvoiceController {
 
     @RequestMapping("/home/index")
-    public String taxInvoiceForm() { // Mapping GET Request and call home-index html page
+    public String taxInvoiceForm(Model model) {
+        ExchangedDocument exchangedDocument = new ExchangedDocument();
+        model.addAttribute("exchangedDocument", exchangedDocument);
         return "home-index";
```

Comment:
ทำไม return "home-index" หล่ะ มันจะ redirect กลับไปหน้า home-index หรอ?

(ทำไมตั้งชื่อหน้าว่า home-index ชื่อไม่สื่อมากๆ)

Reply:
- return เป็นชื่อ html ของหน้านั้นครับ นั่นคือ "home-index"
- ชื่อยังไม่รู้จะตั้งชื่อว่าอะไรครับเลยตั้งไว้ก่อน เดี๋ยวจะเปลี่ยนแน่นอนครับ

Reply:
ok

Reply:
it'd be nice for "templates/controller/action"